bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -0,0 +1,454 @@
+                  }
+                },
+                {
+                  'media': '(min-width: 1024px)',
+                  'srcset': {
+                    'img/firefox/challenge-the-default/hero-image-desktop.png': 'default'
+                  }
+                },
+              ],
+              optional_attributes={
+                'loading': 'eager',
+                'class': 'c-hero-image',
+                'alt': 'Image of a seagull sitting on a no birds allowed sign'
+              }
+          )}}
+        </div>
+      </div>
+    </div>
+  </section>
+
+    <section class="mzp-l-content mzp-t-content-xl c-ctd-features">
+      <h2>Unsere <br>Lieblingsfeatures</h2>
+      <ul class="mzp-l-columns mzp-t-columns-four">
+        {% call picto(
+          base_el='li',
+          title='Nahtloses Browsen',
+          image=resp_img(
+          url='img/firefox/challenge-the-default/cursor.svg',
+          optional_attributes={
+            'class': 'mzp-c-picto-image',
+            'loading': 'lazy'
+            }
+          ),
+          body=True,
+        ) %}
+          <p>Hab offene Tabs und Passwörter griffbereit auf all deinen Geräten – wenn du das willst.</p>
+        {% endcall %}
+        {% call picto(
+          base_el='li',
+          title='Einfaches Multitasking',
+          image=resp_img(
+            url='img/firefox/challenge-the-default/screen.svg',
+            optional_attributes={
+              'class': 'mzp-c-picto-image',
+              'loading': 'lazy'
+            }
+          ),
+          body=True,
+        ) %}
+          <p>Schaue Videos in einem Pop-up-Fenster einfach weiter, während du browst.</p>
+        {% endcall %}
+        {% call picto(
+          base_el='li',
+          title='Non-Profit-Power*',
+          image=resp_img(
+            url='img/firefox/challenge-the-default/toggle.svg',
+            optional_attributes={
+              'class': 'mzp-c-picto-image',
+              'loading': 'lazy'
+            }
+          ),
+          body=True,
+        )%}
+          <p>Firefox wird von der Mozilla Foundation unterstützt und stellt Menschen über Profite.</p>
+          <small>*Wir wissen, dass das kein Feature ist, aber hier machen wir die Regeln. :)</small>
+          </p>
+        {% endcall %}
+        {% call picto(
+          base_el='li',
+          title='Schnelles Bearbeiten von PDFs',
+          image=resp_img(
+            url='img/firefox/challenge-the-default/input.svg',
+            optional_attributes={
+              'class': 'mzp-c-picto-image',
+              'loading': 'lazy'
+            }
+          ),
+          body=True,
+        )%}
+          <p>Bearbeite PDFs ohne Umwege direkt im Browser.</p>
+        {% endcall %}
+      </ul>
+    </section>
+
+    <section class="c-ctd-toggles">
+      <div class="mzp-l-content mzp-l-content mzp-t-content-xl">
+        <div class="toggle-grid small">
```

Comment:
**Nit:** On smaller screens, this section seems to be overflowing:

![Screenshot 2023-06-22 at 1 14 34 PM](https://github.com/mozilla/bedrock/assets/42309026/def76f51-0b53-4d14-aebc-47c512c9c447)

Reply:
done and done!